debian/postinst.d/14-python-packages.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+main() {
+    wget https://bootstrap.pypa.io/get-pip.py && \
+      env python3.9 get-pip.py && \
+      env python3.9 -m pip install -r /usr/share/ottopia-keepalived-exporter/requirements.txt
```

Comment:
mmm .. why in `/usr/share` ?
Where keepalived-exporter is being installed ?

Reply:
5be4750b2f52182d43aacaa82b2b47af05600c32